### ISSUE TITLE
Not all categories show up in legend

### ISSUE THREAD

**Issue by andy-esch** [CartoDB/cartoframes#1131]
A dataset I'm visualizing has some null values but they are not showing up in the legend.

Here is the count of values:
```
N      190963
NaN     10237
Y        3692
Name: vacant, dtype: int64
```
How they appear in the legend
<img width="371" alt="Screen Shot 2019-10-28 at 2 51 59 PM" src="https://user-images.githubusercontent.com/1041056/67708463-97460280-f992-11e9-9ebf-606c2cec8a66.png">

And how they appear in the widget
<img width="289" alt="Screen Shot 2019-10-28 at 2 52 42 PM" src="https://user-images.githubusercontent.com/1041056/67708485-9f9e3d80-f992-11e9-8377-ac3d6b9e791e.png">
(Note: the counts are different from a above because of the viewport)

My expectation is that the legend would show 'null' or 'missing' or 'empty' for the null valued rows.

**Comment by Jesus89** [comment]
https://app.clubhouse.io/cartoteam/story/93694/render-null-values

**Comment by Jesus89** [comment]
This is fixed in the new CF version using WebSDK. It will be available on the next 1.1 release.
